test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3372,6 +3372,48 @@ def forward(self, x, k):
         k = torch.tensor(3)
         self.run_test(MyModuleDynamic(), [x, k])
 
+    @disableScriptTest()
```

Comment:
Can you add a string arg to disableScriptTest or a comment that explains why this test fails in scripting mode?